lib/platform-stats.js

```diff
@@ -166,3 +170,27 @@ export const mapParticipantsToIds = async (pgClient, participantsSet) => {
   assert.strictEqual(created.length, newAddresses.length)
   return ids.concat(created.map(r => r.id))
 }
+
+/**
+ * @param {import('./typings.js').CreatePgClient} createPgClient
+ */
+export const updateTopMeasurementStations = async (createPgClient) => {
```

Comment:
```suggestion
export const updateTopMeasurementParticipants = async (createPgClient) => {
```

... to match the name we use in other places.

Reply:
+1

Reply:
Ah missed that, addressed in https://github.com/filecoin-station/spark-evaluate/pull/278/commits/3b8b7416a86cb7cfe21fd52ca6f45c8c8b8c6f08